test/parallel/test-repl.js

```diff
@@ -142,7 +142,7 @@ function error_test() {
       expect: prompt_unix },
     // But passing the same string to eval() should throw
     { client: client_unix, send: 'eval("function test_func() {")',
-      expect: /\bSyntaxError: Unexpected end of input/ },
+      expect: /^\bSyntaxError: Unexpected end of input/ },
```

Comment:
1. Oh, sorry, I did not think this broke matching.

2. I think that removing the `\b` is safe only if we place `^` before)